mail_tracking_mailgun/models/mail_tracking_email.py

```diff
@@ -203,11 +203,7 @@ def action_manual_check_mailgun(self):
         """
         api_key, api_url, domain, *__ = self._mailgun_values()
         for tracking in self:
```

Comment:
This still can be empty, so do:

```suggestion
        for tracking in self.filtered("message_id"):
```

Reply:
:+1: